spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesInformerDiscoveryClient.java

```diff
@@ -63,13 +63,13 @@ public class KubernetesInformerDiscoveryClient implements DiscoveryClient {
 
 	private static final LogAccessor LOG = new LogAccessor(LogFactory.getLog(KubernetesInformerDiscoveryClient.class));
 
-	private final SharedInformerFactory sharedInformerFactory;
```

Comment:
and this is how the client inners are changing.

Instead of a single `Lister<V1Service>`, we change to `List<Lister<V1Service>>`, and so for all other private fields.